apps/editing-toolkit/editing-toolkit-plugin/global-styles/class-global-styles.php

```diff
@@ -78,6 +78,13 @@ class Global_Styles {
 	 */
 	private $plugin_name = 'jetpack-global-styles';
 
+	/**
+	 * The provider's root URL for retrieving font CSS.
+	 *
+	 * @var string
+	 */
+	private $root_url = 'https://fonts.googleapis.com/css2';
```

Comment:
So I noticed I copied `css2` (V2 API) and replaced `css` (V1 API) [from Jetpack](https://github.com/Automattic/jetpack/blob/b313bd253afdf919adf7a71a866d0ab85b3a7bda/projects/packages/google-fonts-provider/src/class-google-fonts-provider.php#L31).

Everything actually still works, either because we load fonts also in individual calls (which succeed with v2) or because they're cached in my browser. Huge query to load all the fonts in one api call fails, though.

Switching back to v1 API in follow up PR that I'll just merge without waiting: https://github.com/Automattic/wp-calypso/pull/71451